src/medCore/database/medDatabaseModel.cpp

```diff
@@ -28,6 +29,17 @@
 #include <medMetaDataKeys.h>
 #include <medAbstractDatabaseItem.h>
 
+#include <medSettingsManager.h>
+
+
+QString anonymise(const QString name) {
```

Comment:
Why put the opening ```{``` at the end of the function signature, when all the other functions in the file use the other format (with the brace on a new line) ?